doc/Overview/Validation/ValidationOverview.md

```diff
@@ -1,16 +1,158 @@
 ---
 uid: Overview.Validation
 ---
+
 # Validation
 
-Two options:
-- Class can inherit from ObservableValidator and use attributes on properties to define validation rules (eg [Required])
-- Can use [FluentValidation](https://www.nuget.org/packages/FluentValidation/) to execute validation rules. Check out the section on [Dependency Injection](https://docs.fluentvalidation.net/en/latest/di.html#) on how to configure validators
+Uno.Extensions.Validation provides an `IValidator` service which enforces that properties of an object follow a set of characteristics or behaviors represented by [attributes](https://learn.microsoft.com/dotnet/api/system.componentmodel.dataannotations) declaratively applied to them. This is useful for ensuring that data entered by users is valid before it is saved to a database or sent to a web service.
+
+## Service registration
+
+An extension method for `IHostBuilder` is provided to register the `IValidator` service with the DI container.
+
+```csharp
+protected override void OnLaunched(LaunchActivatedEventArgs e){
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host
+            .UseValidation();
+        });
+...
+```
+
+By default, enabling validation will register the default `Validator` type with the DI container. This class implements `IValidator` and provides functionality to execute validation rules on validatable entities.
+
+## Using the validation service
+
+The `IValidator` service is injected into the constructor of view models which specify the `IValidator` service as a dependency. View models that need to validate data can then use the `ValidateAsync` method to execute validation rules on an object. The `ValidateAsync` method returns a `ValueTask<T>` that completes when validation is complete. The `ValueTask` result is an `IEnumerable<ValidationResult>` which contains elements that indicate whether the object is valid or not. 
```

Comment:
```suggestion
The `IValidator` service is injected into the constructor of view models which specifies the `IValidator` service as a dependency. View models that need to validate data can then use the `ValidateAsync` method to execute validation rules on an object. The `ValidateAsync` method returns a `ValueTask<T>` that completes when validation is complete. The `ValueTask` result is an `IEnumerable<ValidationResult>` which contains elements that indicate whether the object is valid or not. 
```

Reply:
In this context, we talk about the 'view model**s**', so specify should be the right word if I understand correctly.